sdk/java/src/test/java/org/symbol/sdk/utils/CharacterMapBuilderTest.java

```diff
@@ -0,0 +1,113 @@
+package org.symbol.sdk.utils;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class CharacterMapBuilderTest {
+
+	@Test
+	public void testCreateEmptyMap() {
+		// Arrange:
+		final CharacterMapBuilder builder = CharacterMapBuilder.create();
+
+		// Act:
+		final Map<Character, Byte> charMap = builder.build();
+
+		// Assert:
+		assertThat("Character map should be empty", charMap.isEmpty());
+	}
+
+	@Test
+	public void testCreateMapWithSingleArbitraryRangeZeroBase() {
+		// Arrange:
+		final CharacterMapBuilder builder = CharacterMapBuilder.create(b -> b.addRange('d', 'f', (byte) 0));
+		final Map<Character, Byte> expectedMap = new HashMap<Character, Byte>() {
+			{
+				put('d', (byte) 0);
+				put('e', (byte) 1);
+				put('f', (byte) 2);
+			}
+		};
+
+		// Act:
+		final Map<Character, Byte> charMap = builder.build();
+
+		// Assert:
+		assertThat("Character map should be empty", charMap, is(expectedMap));
```

Comment:
message is wrong

Reply:
isn't is identity ?

Reply:
(multiple)

Reply:
is `is` checking equals or identiy (same object)?

Reply:
It is more for readability, but I will update them all to use ``equalTo``
```
Sugar

Hamcrest strives to make your tests as readable as possible. For example, the is matcher is a wrapper that doesn’t add any extra behavior to the underlying matcher. The following assertions are all equivalent:

assertThat(theBiscuit, equalTo(myBiscuit)); 
assertThat(theBiscuit, is(equalTo(myBiscuit))); 
assertThat(theBiscuit, is(myBiscuit));

The last form is allowed since is(T value) is overloaded to return is(equalTo(value)).
```

Reply:
ok, in that case, just use either equalTo or is consistently everywhere